### ISSUE TITLE
FuelFactors CO/CO2

### ISSUE THREAD

**Issue by ElliotGlassman** [NREL/EnergyPlus#10358]
Issue overview
--------------
There is an error in calculation emissions... has field for source energy factor which is being used to calculate CO2 in the Annual and Peak Values - Other by Weight/Mass table of.htm.

### Details
Used the FuelFactors as formulated here: https://bigladdersoftware.com/epx/docs/23-1/input-output-reference/input-for-output.html#fuelfactors
![image](https://github.com/NREL/EnergyPlus/assets/155757290/63119d12-5899-4ced-938b-cf32773594ce)

Changed the fields for CO2 Emissions factor. No observed change in output Annual and Peak Values - Other by Weight/Mass table of.htm which did not match my hand calculation. 

Then noticed that  the CO field of the table WAS changing and matched my hand calculation.

![image](https://github.com/NREL/EnergyPlus/assets/155757290/72b73974-5e79-43f1-bf05-fff9b41df755)

Seems like the field for Source energy factor is extraneous and emission factors are use for next GHG. Not sure if the error is in the documentation, the calculation, or in the table output

**Comment by amirroth** [comment]
Feels like I had a hiccup someplace in the PollutionModule refactor. I feel judged. I should probably take a look at this.

**Comment by ElliotGlassman** [comment]
THIS IS A JUDGEMENT FREE ZONE!

**Comment by amirroth** [comment]
I appreciate your lack of judgement. However, I can't reproduce this. I ran two versions of the PurchAirTables.idf (in the testfiles directory) one with the original CO2 factor for electricity of 168 and one with 368 and the results are as expected. Are you using the most recent version of EnergyPlus (23.2)?

**Comment by ElliotGlassman** [comment]
I am using 23.1. But I found the issue. I originally got the text from an earlier version which had extra terms for units of measure (below is a shot from the text I actually used, even though I got the first image from the latest documentation!). That is why the CO2 emissions factors were showing up in the CO2 slot.

![image](https://github.com/NREL/EnergyPlus/assets/155757290/4884266a-745b-4adf-a1e2-be622542a8d3)

Ok, now its my turn to be judged.
